src/Service.Tests/SqlTests/RestApiTests/Delete/DwSqlDeleteApiTest.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.SqlTests.RestApiTests.Delete
+{
+    /// <summary>
+    /// Test REST Apis validating expected results are obtained.
+    /// </summary>
+    [TestClass, TestCategory(TestCategory.DWSQL)]
+    public class DwSqlDeleteApiTests : DeleteApiTestBase
+    {
+        private static Dictionary<string, string> _queryMap = new()
+        {
+            {
+                "DeleteOneWithStoredProcedureTest",
+                $"SELECT [id] FROM { _integrationTableName } " +
+                $"WHERE id = 14"
+            }
+        };
+        #region Test Fixture Setup
+
+        /// <summary>
+        /// Sets up test fixture for class, only to be run once per test run, as defined by
+        /// MSTest decorator.
+        /// </summary>
+        [ClassInitialize]
+        public static async Task SetupAsync(TestContext context)
+        {
+            DatabaseEngine = TestCategory.DWSQL;
+            await InitializeTestFixture();
+        }
+
+        /// <summary>
+        /// Runs after every test to reset the database state
+        /// </summary>
+        [TestCleanup]
+        public async Task TestCleanup()
+        {
+            await ResetDbStateAsync();
+        }
+
+        #endregion
+
+        [TestMethod]
+        public async Task DeleteOneInViewBadRequestTest()
+        {
+            string expectedErrorMessage = $"View or function '{_defaultSchemaName}.{_composite_subset_bookPub}' is not updatable " +
+                                           "because the modification affects multiple base tables.";
+            await base.DeleteOneInViewBadRequestTest(expectedErrorMessage);
+        }
```

Comment:
was this just a gap for the 'delete' case in testing REST DW? or do you also need to ensure DW view tests are added for find/insert/update?

Reply:
We didnt have delete tests for rest dw. I just added whatever coverage we have for the other db's.

Reply:
We have one test for insert. Since the format we are using is insertTest will inherit from the base - this means all base tests are run. For DW, since insert does not return the resultant rows (we went with a DBOperationResult return type in GQL) Have to see similar for REST (how do we handle no data being returned from a mutation in rest scenario). Hence I am removing the insert test for now and will handle it once I add the logic for mutation support all up for dw for rest scenario. this should atleast unblock this pr and add required stored proc support